src/molecules/form/index.jsx

```diff
@@ -2,31 +2,44 @@ import { StyledFieldset } from "../../ions/styles";
 import Input from "../../atoms/input";
 import Button from "../../atoms/button";
 import React, { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useStore from "../../ions/useStore";
 
 const Form = () => {
-	const [value, setValue] = useState("");
-	const [name, setName] = useState("");
+	const name = useStore(state => state.name);
+	const setName = useStore(state => state.setName);
+	const router = useRouter();
+	const handleSubmit = () => {
+		router.push({
+			pathname: "/lobby/195",
+		});
+	};
+
+	//console.log(router);
+	//return null;
```

Comment:
you probably don't need this anymore. :)

Reply:
Thanks for the comments! I deleted it, commit incoming :-)